packages/connect-extension-protocol/src/index.ts

```diff
@@ -101,18 +89,10 @@ export interface ProviderMessageData {
   /** What action the `ExtensionMessageRouter` should take **/
   action: "forward" | "connect" | "disconnect"
   /** The message the `ExtensionMessageRouter` should forward to the background **/
-  message?: MessageToManager
-}
-
-/**
- * The message from the `ExtensionProvider` in the app that is intended to be
- * sent on to the manager in the extension background.
- */
-export interface MessageToManager {
   /** Type of the message. Defines how to interpret the {@link payload} */
-  type: "rpc" | "spec"
+  type?: "rpc" | "spec"
```

Comment:
Why is the `type` now optional? :thinking:

Reply:
`message?: MessageToManager` was optional. Now that the interfaces are merged, as a result these are optional